cfn.yaml

```diff
@@ -0,0 +1,114 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This job runs regularly to pull data from presto and drop it in an email to the team.
+Parameters:
+  Stack:
+    Description: Stack name
+    Type: String
+    Default: supporter-metrics-job
+  App:
+    Description: Application name
+    Type: String
+    Default: supporter-metrics-job
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - CODE
+      - PROD
+    Default: CODE
+  DeployBucket:
+    Description: Bucket where RiffRaff uploads artifacts on deploy
+    Type: String
+    Default: supporter-metrics-job-dist
+  EmailFrom:
+    Description: from address for emails
+    Type: String
+    Default: john.duffell@guardian.co.uk
+  EmailTo:
+    Description: email address to send emails to
+    Type: String
+    Default: john.duffell@guardian.co.uk
+  PrestoUrl:
+    Description: presto db jdbc url
+    Type: String
+    Default: jdbc:presto://...
+  EmailLine:
+    Description: email address to send emails to
+    Type: String
+    Default: See https://drive.google.com (TODO) for more details
+Resources:
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: logs
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - logs:CreateLogGroup
+                - logs:CreateLogStream
+                - logs:PutLogEvents
+              Resource: arn:aws:logs:*:*:*
+        - PolicyName: lambda
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - lambda:InvokeFunction
+              Resource: "*"
+        - PolicyName: email
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - ses:SendEmail
+              Resource: "*"
+  Lambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub ${App}-${Stage}
+      Code:
+        S3Bucket:
+          Ref: DeployBucket
+        S3Key: !Sub ${Stack}/${Stage}/${App}/${App}.jar
+      Environment:
+        Variables:
+          Stage: !Ref Stage
+          Stack: !Ref Stack
+          App: !Ref App
+          EmailFrom: !Ref EmailFrom
+          EmailTo: !Ref EmailTo
+          PrestoUrl: !Ref PrestoUrl
+          EmailLine: !Ref EmailLine
```

Comment:
this way of passing private/config to the lambda is only sussatinable for small numbers of params.  We might need an S3 file if/when it gets too much